docker/base-images.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Google LLC
```

Comment:
I think you want this file to at docker/gcb/base-images.yaml (that's where the current GCB config is picking it up)

Reply:
Noted I'll check it in there.